jobsdb/jobsdb.go

```diff
@@ -775,7 +775,7 @@ func (jd *Handle) init() {
 	}
 
 	if string(jd.conf.payloadColumnType) == "" {
-		jd.conf.payloadColumnType = payloadColumnType(jd.config.GetStringVar(string(JSONB), jd.configKeys("payloadColumnType")...))
+		jd.conf.payloadColumnType = TEXT
```

Comment:
we don't need `jd.conf.payloadColumnType` anymore
```suggestion
```

Reply:
It makes our life easier while testing. For example tests where we run a jobsdb with one column type and restart it with another.
Can be removed eventually when we remove the handling for type interchanging.